test/functional/wallet_importmulti.py

```diff
@@ -458,6 +468,74 @@ def run_test (self):
                 "timestamp": "",
             }])
 
+        # Import P2WPKH address
```

Comment:
There are a number more scenarios that make sense to test, I think:
* Address based import (without key/script) of P2WSH-multisig
* For all of P2WPKH, P2WSH-multisig, P2SH-P2WPKH, P2SH-P2WSH-multisig, versions with script/pubkey but no private key, and test that the result is solvable.
* P2WPKH with private key, result must be spendable (not just watchonly)
* P2WSH-multisig and P2SH-multisig with more than 1 key, where some of the keys are specified as private keys, and some as public keys.

Reply:
@Sjors Correct.